recipe/meta.yaml

```diff
@@ -1,4 +1,4 @@
-{% set version = "0.9.6" %}
+{% set version = "1.0.0a4" %}
```

Comment:
Another pre-release. Ping @CJ-Wright

Reply:
Is there a definitive pre-release format?

Reply:
For Python, there is [PEP 440]( https://www.python.org/dev/peps/pep-0440/ ). Though that definitely doesn't cover all version schemes in all languages. That said, this would be a good start. If you are checking the GitHub API, you could also inspect if a release was marked as prerelease.

Reply:
Not really. Sometimes people add it to another label.

The issue lies with how conda manage labels, that won't behave like `pip --pre`, conda will get all the packages from the label instead of only the package the user requested (the label behaves like a channel).

@msarahan are there plans to create a similar functionality like `pip --pre` to conda? (Or maybe that already exists and I am outdated?)

Reply:
So this seems to be an oversight in the autoticker.  There actually already is a `1.0.0a4` package uploaded to anaconda.org, see https://anaconda.org/conda-forge/github3.py/files?sort=basename&sort_order=desc

The version number here was set back to 0.9.6 to obtain a package for a release that was skipped. It seems like the autoticker is assuming that what is in the meta.yaml is the most recent version that is available. But that is not actually true. The most recent version available has to be checked from anaconda.org

Reply:
Is there an api for aonaconda.org?

Reply:
Issue raised here: https://github.com/regro/cf-graph/issues/46